### PR TITLE
Add start_tls, remove bind_password

### PR DESCRIPTION
**Pull request by harry-kim**
Refs: #SDE-3474

TODO:
- [x] Set milestone label in pull request (SDE version)
- [x] Updated changelog.md
- [x] Completed documentation for the new endpoint.
- [x] Feature has been merged into SDE develop.

### CODE REVIEW

**Comment by lmontopo** [comment]
Looks good +1.

File: source/includes/connections.md [hunk 1]
```diff
@@ -157,7 +157,8 @@ Authorization: Token "YOUR SDE ACCESS TOKEN"
 		"group_query": "(objectClass=group)",
 		"bind_dn": "cn=Administrator,cn=Users,dc=example,dc=org",
 		"base_dn": "dc=example,dc=org",
-		"bind_password": "pass"
+		"bind_password": "pass",
+        "start_tls": true
```
Comment (aeldamaty) [inline source/includes/connections.md]: line this up properly

**Comment by aeldamaty** [comment]
One comment. Then +1

**Comment by lmontopo** [comment]
my +1 remains

**Comment by aeldamaty** [comment]
+2.

**Pull request merged**